src/fromager/dependencies.py

```diff
@@ -119,6 +119,13 @@ def get_build_backend_dependencies(
     )
     deps = _filter_requirements(req, orig_deps)
 
+    # Virtual envs of Python 3.11 and 3.12 come with older pip that do
+    # not support --config-settings.
+    if pbi.config_settings and req.name != "pip":
```

Comment:
Is there a reason we wouldn't want to always do this? Why make the user configure it?

Reply:
Actually we don't need this code. I forgot that Fromager uses 3rd party `virtualenv` instead of stdlib `venv` to create virtual envs. The package ships with latest pip, https://virtualenv.pypa.io/en/latest/changelog.html